src/app/Main/Body/QueryPage/Results/QueryPage-ResultData.component.tsx

```diff
@@ -90,36 +103,49 @@ export function QueryPageTableRestultComponent(
     );
 
     return (
-        <div css={dataStyle}>
-            <TableMainHeaderComponent
-                isTitle={props.displayRowCol.isCol}
-                isRow={false}
-                height={tableHeight}
-                width={headerWidth}
-                text={colMainHeader}
-            />
-            <div css={tableDivStyle}>
-                <TableMainHeaderComponent
-                    isTitle={props.displayRowCol.isRow}
-                    isRow
-                    height={tableHeight}
-                    width={headerWidth}
-                    text={rowMainHeader}
-                />
-                <div
-                    css={dataTableStyle}
-                    ref={(node: HTMLElement | null) => div(node, "totalWidth")}
-                >
-                    <ResultsTable
-                        allIsolates={table.statisticData}
-                        isIsolates={isIsolates}
-                        columnAttributes={props.columnAttributes}
-                        getSize={div}
-                        isRowNotCol={isRowNotCol}
-                        isRowAndCol={isRowAndCol}
+        <Accordion defaultExpanded>
+            <AccordionSummary
+                expandIcon={<ExpandMoreIcon />}
+                aria-controls="table-accordion-content"
+                id="table-accordion-header"
+            >
+                <h3 css={subHeadingStyle}>{accordionHeader}</h3>

```

Comment:
- consider using h# tags only for document like content and in the correct order h1>h2>h3
- consider using explicit shared styles for elements like the accordion title